community/modules/scheduler/htcondor-access-point/outputs.tf

```diff
@@ -14,7 +14,12 @@
  * limitations under the License.
  */
 
-output "list_instances_command" {
```

Comment:
Should we keep this output (in addition to the 2 new ones) to avoid breaking existing blueprints?

Reply:
This is landing on a `htcondor-in-flight` branch I'm deliberately using to stage things before getting to `develop`. So it's not even on develop, much less a release.

Reply:
Ok, makes sense!